gdraw/gfilechooser.c

```diff
@@ -138,6 +138,9 @@ return( NULL );
 return( NULL );
 	++pattern;
     }
+    if (pattern > eop) {
+        return NULL;
```

Comment:
What was happening here?

Reply:
This code is super buggy. I was having trouble getting it to explicitly support both `woff` and `woff2. Really, `pattern` should never run past `eop` (end of pattern). This is just good enough to get it working.